generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs

```diff
@@ -1979,6 +1979,6 @@
     "waitIncrementSeconds": "60",
     "offlineSessionMaxLifespanEnabled": "false"
   },
-  "keycloakVersion": "6.0.1",
+  "keycloakVersion": "7.0.0",
```

Comment:
are there no structural changes in this upgrade?

Reply:
You can import the existing json and then export to check difference in configurations